media-api/conf/example-image.json

```diff
@@ -0,0 +1,160 @@
+{
```

Comment:
any reason why we're not fetching this from elasticsearch?

Reply:
The idea was to have the same image in all environments without having to worry whether it exists in ES or whether the ids are different for example.

Reply:
makes sense 👍 however, eyevine would need to be able to download the image to fully test the process their end. Could we return data for a real image, with links to our cdn etc?

Reply:
Will do, makes sense.